### ISSUE TITLE
Scene Loading

### ISSUE THREAD

**Issue by JenkinJoe** [cgarciae/karma#2]
Hello there,

there is another question that came up to my mind recently.
How do you determine whether a scene (another view so to speak) is fully loaded asynchronously (e.g. while in loading screen)? I know that there is an app.ready property returning boolean. 
But how do you exactly switch to another level (view instance) if it's fully loaded.

Could you please make an example for that one?
Also interesting to know would be how you usually share objects between different scenes.

With best regards,
Joe

**Comment by cgarciae** [comment]
@JenkinJoe 
Yeah, that boolean was kind of experimental and its currently not being used but I can easily tweak the middleware to take it into account. What exactly is your current problem? Karma doesn't delete te current  view until the prefab for the next view is loaded, so your problem might actually not exist.

I've faced issues when a view needs to fetch something from the server and it takes time, how I currently solve this problem is just putting a "Loading" UI Image on that view and turn it off when the view is ready.
